### PR TITLE
Create Updated main.py

### PR DESCRIPTION
**Pull request by shaypheraddison**
Is this the kind of idea for creating a function? Or can it be simplified even further?

### CODE REVIEW

**Review by resoluteCoder — COMMENTED** [review]
I think this a great start and exactly what I was referring to. 

Let's do the same with accepting the input from the user that also returns the conversion in the proper type. Then store the two in separate variables which then get passed to the calc function

**Review by shaypheraddison — COMMENTED** [review]
Is this what you were saying? I feel like its a little excessive and don't know how to consolidate this much

**Review by resoluteCoder — COMMENTED** [review]
I like the addition of the list. Now you should be able to loop over each item and use the item in the print state of the input. This way you have one input statement but this will also require having to change how you declared/init your variables.

What you could do is look into dictionaries and use a single dict for all the values.

**Comment by shaypheraddison** [comment]
I think I broke it. I had it working through the loop at one point today. it was even converting it to a float number, but now I keep getting an error when I have either float or int in front of the input. the error shows up after entering the first number too.

File: Updated main.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+def get_user_input():
```
Comment (resoluteCoder) [inline Updated main.py:1]: this code can be condensed to:
```python
stats = {}
labels = ["hits", "at-bats"] #the rest ...
for label in labels:
    str_value = input(f'Enter the number of {label}')
    stats[label] = int(str_value)
return stats
```

File: Updated main.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        return 0.0
+    else:
+        return total_bases / at_bats
+
+def calculate_ops(hits, walks, hit_by_pitch, at_bats, sacrifice_flies, singles, doubles, triples, home_runs):
+    obp = calculate_on_base_percentage(hits, walks, hit_by_pitch, at_bats, sacrifice_flies)
+    slg = calculate_slugging_percentage(singles, doubles, triples, home_runs, at_bats)
+    return obp + slg
+
+def calculate_and_display_slash_line(stats):
```
Comment (resoluteCoder) [inline Updated main.py]: Let's change this from doing two things. Remember you want functions to do one thing (if you can help it)

try return the calculations in a format that can be sent do a `display_slash_line` function

**Review by resoluteCoder — CHANGES_REQUESTED** [review]
Very nicely done a couple of improvements but otherwise very impressed! Let's keep it up!!

**Pull request merged**